### PR TITLE
webtasks added to project

### PR DESCRIPTION
**Pull request by gjstoychev**
Task 1, 2 and 3 are added. I am going to work on Task 2 and use fetch instead of jquerry

### CODE REVIEW

File: task3.html [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+            <textarea id="address" rows="10" cols="30"></textarea><br>
+            <button type="button" onclick="checkInput()">Check Input</button>
+            <br>
+            <p id="demo"></p>
+        </form>
+    </fieldset>
+
+    <script>
+        function checkInput() {
+            var firstname = {   name: "firstname",
```
Comment (mgenov) [inline task3.html]: name of the fields becomes duplicated: 

```javascript
var fields = [  
   { name: "firstname", .... },
   { name: "lastname", .... },
   { name: "egn", .... }
]
```

File: task3.html [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+                                span: "egnSpan"
+                            },
+                age =       {   name: "age",
+                                value:document.getElementById("age").value,
+                                regexp:/^[0-9]{2,3}$/,
+                                errorMsg:"must be a number between 18 and 118",
+                                span: "ageSpan"
+                            },
+                address =   {   name: "address",
+                                value:document.getElementById("address").value,
```
Comment (mgenov) [inline task3.html]: The value could be taken by the name of the field instead to be duplicating it in each field. 

```javascript
for (var f in fields) {
  var value = document.getElementById(f.name).value
  ...
  
}
```

File: task3.html [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+            width: 100%;
+        }
+        span {
+            font-size: 12px;
+        }
+    </style>
+</head>
+
+<body>
+    <fieldset>
```
Comment (mgenov) [inline task3.html]: Bootstrap has form fields which could be used instead: 

http://getbootstrap.com/css/#forms
```html
<form>
  <div class="form-group">
    <label for="exampleInputEmail1">Email address</label>
    <input type="email" class="form-control" id="exampleInputEmail1" placeholder="Email">
  </div>
  <div class="form-group">
    <label for="exampleInputPassword1">Password</label>
    <input type="password" class="form-control" id="exampleInputPassword1" placeholder="Password">
  </div>
  ...

</form>
```

**Comment by mgenov** [comment]
You need to squash your commits when PR is ready for final review.

File: task3.html (hunk 3 above)
Comment (gjstoychev) [inline task3.html]: My bad I did it from scratch instead of follow the classic way with label for each input. Fixed.

File: task3.html (hunk 1 above)
Comment (gjstoychev) [inline task3.html]: Thank you. Fixed.

File: task3.html (hunk 2 above)
Comment (gjstoychev) [inline task3.html]: Thank you. Fixed.

**Comment by mgenov** [comment]
Thinks are looking fine. When you squash your commits I'll be able to merge.

**Pull request merged**